com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner.cs

```diff
@@ -272,21 +272,21 @@ private Vector3 ConfinePoint(Vector3 camPos)
         }
 
         // Camera must be orthographic
-        private Vector3 ConfineScreenEdges(CinemachineVirtualCameraBase vcam, ref CameraState state)
+        Vector3 ConfineScreenEdges(ref CameraState state)
         {
-            Quaternion rot = Quaternion.Inverse(state.CorrectedOrientation);
```

Comment:
This is the fix:
- We need the rotation, not the inverse.

The rest is just cosmetic or unused.